### ISSUE TITLE
Add ms in log

### ISSUE THREAD

**Issue by rakita** [openethereum/openethereum#326]
For better logs, it should be good to see milliseconds in there.

**Comment by tbcd** [comment]
Agreed, although it might be better as an opt-in flag to avoid breaking existing monitoring scripts that parse these logs. Happy to submit a PR for this if you like, as I already use it locally.

Also added a flag to always use UTC timestamps, but that may not be as generally useful to others.
